### ISSUE TITLE
 get joint coordinate

### ISSUE THREAD

**Issue by bianxu123** [ildoonet/tf-pose-estimation#410]
how can I get the joint coordinate from a image or a video

**Comment by sprkshnarayan** [comment]
yeah exactly! i need the coordinates of the joints from the web-cam video or image. @ildoonet can u please help us out here and tell the file in which these coordinates are stored during runtime.

**Comment by eehoeskrap** [comment]
This code is the draw_humans part of the estimator.py.
Add 3 line.

flat = [0.0 for i in range(36)]
...
flat[i*2] = center[0]
flat[i*2+1] = center[1]

flat is the set of joint coordinate.

--------------------------------------------------------------------------------------------------------------
			for i in range(common.CocoPart.Background.value):
				if i not in human.body_parts.keys():
					continue

				body_part = human.body_parts[i]
				#print(body_part)
				center = (int(body_part.x * image_w + 0.5), int(body_part.y * image_h + 0.5))
				centers[i] = center

				#add x coordinate
				flat[i*2] = center[0]
				#add y coordinate
				flat[i*2+1] = center[1]

**Comment by DilaraSina** [comment]
Thank you, do you know which indices belong which joint?

**Comment by eehoeskrap** [comment]
here :)

tf-pose-estimation/tf_pose/common.py

class CocoPart(Enum):
    Nose = 0
    Neck = 1
    RShoulder = 2
    RElbow = 3
    RWrist = 4
    LShoulder = 5
    LElbow = 6
    LWrist = 7
    RHip = 8
    RKnee = 9
    RAnkle = 10
    LHip = 11
    LKnee = 12
    LAnkle = 13
    REye = 14
    LEye = 15
    REar = 16
    LEar = 17
    Background = 18

**Comment by JuanPablovas89** [comment]
Thanks a lot eehoeskrap.! Right now I try to run the this in the command prompt "python run.py --model=mobilenet_thin --resize=432x368 --image=./images/p1.jpg" and then I obtain the results pose results of one image, However, I wonder how can I obtain results for several images, and save them as .jpg. Any suggestion?

**Comment by DilaraSina** [comment]
thank you @eehoeskrap

**Comment by DungLai** [comment]
> Thanks a lot eehoeskrap.! Right now I try to run the this in the command prompt "python run.py --model=mobilenet_thin --resize=432x368 --image=./images/p1.jpg" and then I obtain the results pose results of one image, However, I wonder how can I obtain results for several images, and save them as .jpg. Any suggestion?

Create an openCV script

**Comment by DungLai** [comment]
![numbering](https://user-images.githubusercontent.com/27338301/55205287-59c79e00-5226-11e9-8ec8-28f7c760ddc8.PNG)
